packages/documentation-site/patternfly-docs/content/accessibility/helper-text/helper-text.md

```diff
@@ -3,50 +3,84 @@ id: Helper text
-        This will will not be announced since it is not linked.
-      </HelperTextItem>
-    </HelperText>
-    ```
-- Pass in the `isLiveRegion` prop to the HelperText component when you expect or intend for any helper text items to dynamically update in some way. This can include text that only renders in a specific situation, or when the text itself will change.
-    - Only content within the accessibility tree will be announced when updated. CSS updates and updates to text that is hidden via the `aria-hidden` attribute (such as any helper text icons) will not be announced.
-
-The following props/attributes have been added for you or are customizable in PatternFly:
-
-| React prop/attribute | React component that it should be applied to | Which HTML element it appears on in markup | Reason used |
-| -- | -- | -- | -- |
-| `aria-describedby` | TextInput or another form component |  | Used to pass in one or more `id`'s as a string, allowing assistive technologies to announce the helper text content to users. |
-| `id` | HelperText \| HelperTextItem | .pf-v5-c-helper-text \| .pf-v5-c-helper-text__item | Used to pass into a form component's `aria-describedby` prop. |
-| `isLiveRegion` | HelperText | .pf-v5-c-helper-text | Creates an `aria-live="polite"` live region, which allows assistive technologies to announce content updates within it. |
\ No newline at end of file
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
+
+## Accessibility
+
+To implement an accessible PatternFly **helper text**:
+
+- Give the helper text item an `id`, and pass that `id` to the main element's `aria-describedby` attribute. Alternatively if there are multiple helper text items, the helper text component can be given the `id` to be passed to the main element's `aria-describedby`. For example, the following code block includes two valid examples of passing `id` values to a main element:
+  ```noLive
+  <TextInput aria-describedby="helper-text-1" />
+  <HelperText>
+    <HelperTextItem id="helper-text-1">
+      Some helper text content
+    </HelperTextItem>
+  </HelperText>
+
+  <TextInput aria-describedby="helper-text-2" />
+  <HelperText id="helper-text-2" component="ul">
+    <HelperTextItem component="li">
+      Some helper text content
+    </HelperTextItem>
+    <HelperTextItem component="li">
+      Some other helper text content
+    </HelperTextItem>
+  </HelperText>
+  ```
+- Structure the helper text as a `ul` element and each helper text item as a `li` element if there is more than one helper text item.
+- Make the helper text component an `aria-live` region if the helper text item(s) have static text and styling, but will dynamically render.
+- Ensure there is visually hidden text for assistive technologies such as screen readers that conveys the status type if the helper text item has static text and will always be rendered, but the styling will dynamically update.
+- Avoid passing interactive content as helper text.
```

Comment:
Currently [helper text design guidelines](https://www.patternfly.org/components/helper-text/design-guidelines#content-considerations) mention that a link to additional info can be provided, so maybe some input from @andrew-ronaldson or @mmenestr would be helpful here. Is that still an intended use case for a helper text? Just wondering whether helper text should include content that has to be navigated to on another page, or if it should only include content that is immediately relevant for a user.

Reply:
Hmm to be honest, I'd say not. I might edit the guidelines to delete that bullet because I don't think it makes sense. It only makes sense to add a link to external information in a popover!

Reply:
I can remove that bullet from the design guidelines as part of this PR if you'd like?

Reply:
@thatblindgeye go for it